app/renderer/js/utils/proxy-util.ts

```diff
@@ -121,7 +121,8 @@ class ProxyUtil {
 			return socksString;
 		})();
 
-		Promise.all([httpProxy, httpsProxy, ftpProxy, socksProxy]).then(values => {
+		(async () => {
```

Comment:
Instead of adding an `async` lambda here, we should make the entire `resolveSystemProxy` function `async`.